bucket/picotorrent.json

```
@@ -25,12 +41,32 @@
     "bin": "PicoTorrent.exe",
     "pre_install": [
         "if (!(Test-Path \"$persist_dir\\Session.dat\")) { Set-Content \"$dir\\Session.dat\" $null -Encoding Ascii }",
-        "if (!(Test-Path \"$persist_dir\\PicoTorrent.json\")) { Set-Content \"$dir\\PicoTorrent.json\" '{}' -Encoding Ascii }"
+        "if (!(Test-Path \"$persist_dir\\PicoTorrent.json\")) { Set-Content \"$dir\\PicoTorrent.json\" '{}' -Encoding Ascii }",
+        "if (!(Test-Path \"$persist_dir\\PicoTorrent.sqlite\")) { New-Item \"$dir\\PicoTorrent.sqlite\" -ItemType File | Out-Null }"
     ],
     "persist": [
         "Torrents",
         "Session.dat",
-        "PicoTorrent.json"
+        "PicoTorrent.json",
+        "PicoTorrent.sqlite"
+    ],
+    "post_install": [
+        "if (Test-Path \"$dir\\picotorrent-install-associations.reg\") {",
+        "  $picopath = \"$dir\\PicoTorrent.exe\".Replace('\\', '\\\\')",
```

indent with 4 spaces ...

```suggestion
        "    $picopath = \"$dir\\PicoTorrent.exe\".Replace('\\', '\\\\')",
```